haskell/gtk-gui/package.yaml

```diff
@@ -0,0 +1,26 @@
+name:                gtk-gui
+version:             0.1.0.0
+github:              "stackbuilders/blog-code"
+license:             BSD-3-Clause
+author:              "Mark Karpov"
+maintainer:          "jgalarza@stackbuilders.com"
+copyright:           "2016 Mark Karpov"
+
+extra-source-files:
+- README.md
+
+description:         Please see the README on GitHub at <https://github.com/githubuser/gtk-gui#readme>
+
+dependencies:
+- base >= 4.7 && < 5
+- gtk3 >= 0.15.8
```

Comment:
We should restrict the upper bounder so we don't have unexpected breaking changes.